### PR TITLE
changes based on the results of static code analysis by Findbugs

### PR DESCRIPTION
**Pull request by dk2k**
As for removed Assert - its condition is always true, can be checked in Intellij Idea as well

### CODE REVIEW

**Comment by dk2k** [comment]
What do I do now to restart checks?

File: src/main/java/org/springframework/data/jpa/repository/config/JpaRepositoryConfigExtension.java [hunk 1]
```diff
@@ -234,9 +237,17 @@ public void registerBeansForRoot(BeanDefinitionRegistry registry, RepositoryConf
 	@Override
 	protected ClassLoader getConfigurationInspectionClassLoader(ResourceLoader loader) {
 
-		ClassLoader classLoader = loader.getClassLoader();
-
-		return classLoader != null && LazyJvmAgent.isActive(loader.getClassLoader())
+        AtomicReference<ClassLoader> classLoaderRef = new AtomicReference<>();
+        AccessController.doPrivileged(new PrivilegedAction() {
```
Comment (mp911de) [inline src/main/java/org/springframework/data/jpa/repository/config/JpaRepositoryConfigExtension.java:241]: `AccessController` is subject to deprecation and future removal with the deprecation of Java's Security Manager.

Comment (dk2k) [reply src/main/java/org/springframework/data/jpa/repository/config/JpaRepositoryConfigExtension.java:241]: will revert related changes then

**Comment by schauder** [comment]
Thanks, that's merged.

**Pull request closed without merging**